Added IFTTT, Basic command API (mainly for use with IFTTT), a Close All Doors button, and changed alert messaging with 0 time to wait config

ijaron opened the pull request:
IFTTT updates are sent via maker channel. Currently sent on open and close for all doors to the same event code with value1 sent with the door name and value2 being the status. (might recode to use a method more like openhab with an event per door and/or per state but also don't want to make the config too messy)

API can be used via IFTTT webhooks (or anything else that can make get requests) to send a command to the controller (eg `http://public ip or dns/api?key=<key you set>&command=<open, close, or toggle>&id=<door id or all_doors>`)
Key is a string you set in the config.json, I recommend using something like https://www.allkeysgenerator.com/Random/Security-Encryption-Key-Generator.aspx to generate a random 256bit or higher key.

Close all button will close all doors in the open state, all other states are ignored.
![Screenshot_20200612-162450](https://user-images.githubusercontent.com/5156472/84547697-ad0dfb00-acc9-11ea-8175-d8ec2e5f38ba.png)

Also changed message on alerts sent if time to wait is 0. 
Message changed to "%s's garage door just opened"

andrewshilliday commented:
Can you please add a section to the README file that covers how to use the IFTTT integration?  You can probably just copy/paste much from the comment above.  Thanks for the new feature!

ijaron commented:
Updated README and updated IFTTT to use an event per door and state to improve usability if you only want the event(s) to be fired on close/open or only on one/some door(s)
